bots/cloudtrail_enable_log_file_validation.py

```diff
@@ -0,0 +1,27 @@
+"""
+## cloudtrail_enable_log_file_validation
+What it does: Enable log file validation in cloudTrail
+Usage: AUTO: cloudtrail_enable_log_file_validation
+Limitations: None
+"""
```

Comment:
6 files change in this PR.. something went wrong, lets do another quick session to fix everything if you are having problems